front/src/modules/rollingStock/components/RollingStockSelector/RollingStockModal.tsx

```diff
@@ -5,11 +5,13 @@ import { useTranslation } from 'react-i18next';
 
 import { LightRollingStockWithLiveries } from 'common/api/osrdEditoastApi';
 import { enhancedEditoastApi } from 'common/api/enhancedEditoastApi';
-import { getRollingStockID } from 'reducers/osrdconf/selectors';
 import Loader from 'common/Loader';
 import ModalBodySNCF from 'common/BootstrapSNCF/ModalSNCF/ModalBodySNCF';
 import { ModalContext } from 'common/BootstrapSNCF/ModalSNCF/ModalProvider';
 import RollingStockCard from 'modules/rollingStock/components/RollingStockCard/RollingStockCard';
+import { useOsrdSelectors } from 'common/osrdContext';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
```

Reply:
Done